pandas/io/formats/style.py

```diff
@@ -537,6 +538,19 @@ def set_tooltips(
         storage_options=_shared_docs["storage_options"],
         storage_options_versionadded="1.5.0",
         extra_parameters="",
+        extra_examples=textwrap.dedent(
+            """\
```

Comment:
```suggestion
            """
```

Maybe this is causing your docstring error - is this a reason you needed to add this?

Reply:
We wrote it like this because it was present in other docstrings, so we followed that example, I'll add a commit with the change and see If the error stands, thanks.